src/forms/StudyFormSteps.js

```diff
@@ -195,14 +244,19 @@ export const ExternalStep = ({
 };
 
 export const GrantStep = ({
+  newStudy,
```

Comment:
Because new study view and study info view are calling the same `NewStudyForm` component. `newStudy` is a prop used to conditionally render different field and look for creating new study or showing/editing existing study info.

Here is how the form is called from new study view with `newStudy` prop set to `true`
```
<NewStudyForm
    newStudy
    submitValue={submitValue}
    apiErrors={newStudyError}
    history={history}
/>
```

Reply:
instead of having it as a boolean prop could we maybe just compute this internally by looking at the `initialValues`

```js
const initialValues = prefillValues
    ? prefillValues
    : {
        externalId: null,
        name: null,
        shortName: null,
        description: null,
        releaseDate: null,
        version: null,
        attribution: null,
        anticipatedSamples: 0,
        awardeeOrganization: null,
      };
const newStudy = Object.values(intiialValues).some(val => Boolean(val) === true);
```

I _think_ this way if any of the values are prefilled it makes the assumption that this is an existing study.